frontend/pages/admin/TeamManagementPage/TeamDetailsWrapper/TeamSettings/TeamSettings.tsx

```diff
@@ -0,0 +1,208 @@
+import Button from "components/buttons/Button";
+// @ts-ignore
+import InputField from "components/forms/fields/InputField";
+import useTeamIdParam from "hooks/useTeamIdParam";
+import { ITeamConfig } from "interfaces/team";
+import { ITeamSubnavProps } from "interfaces/team_subnav";
+import React, { useCallback, useContext, useEffect, useState } from "react";
```

Comment:
It'd be nice to order the imports similar to other pages (but not a blocker). FWIW, I usually start with react, then third-party components, then out context, hooks, interfaces, services, components, and then relative imports. New lines between each of the groups. All loosely following alphabetical order of the first named input. That's just what looks the cleanest to me.